k8s-install/scripts/install_cni_test.go

```diff
@@ -39,6 +39,7 @@ func runCniContainer(extraArgs ...string) error {
 		"-e", "SLEEP=false",
 		"-e", "KUBERNETES_SERVICE_HOST=127.0.0.1",
 		"-e", "KUBERNETES_SERVICE_PORT=8080",
+		"-v", cwd + ":/template",
```

Comment:
Hmm, so files under `cwd + "/tmp"` will be mapped into the container in two different places.  Does that work?  Does it matter?  It feels slightly inelegant to me; could you introduce another subdirectory of `cwd`, and only map that subdir to `/template`?

Reply:
Moved to a subdir